docs/using.rst

```diff
@@ -413,11 +421,15 @@ to, `install Docker`_, then issue the following command:
    sudo docker run -it --rm -p 443:443 -p 80:80 --name certbot \
                -v "/etc/letsencrypt:/etc/letsencrypt" \
                -v "/var/lib/letsencrypt:/var/lib/letsencrypt" \
-               quay.io/letsencrypt/letsencrypt:latest auth
+               quay.io/letsencrypt/letsencrypt:latest certonly
+
+Certbot will obtain a certificate and place it in the directory
+``/etc/letsencrypt/live`` on your system. 
+You must use the ``certonly`` command
+to install the certificate. 
```

Comment:
Like `auth`, `certonly` doesn't install your certificates to your web server.

To flesh out this distinction a little more, the default behavior of Certbot is the `run` subcommand. This is used if the user types `certbot run` or even just `certbot` because the default subcommand is `run`. What `run` does is prove to the Let's Encrypt certificate authority that you own the domain name(s) you requested, downloads the certificates issued by Let's Encrypt, and then installs them in your webserver.

`certonly` on the other hand omits this last step. `certbot certonly` will prove ownership and get certificates from Let's Encrypt and save them on your computer. After doing that though, the user has to install them to their webserver software themselves.

Assuming you're wanting to explain the use of `certonly`, I think we may be able to just combine this and the previous sentence to say:

"Running Certbot with the ``certonly`` subcommand will obtain a certificate and place it in the directory ``/etc/letsencrypt/live`` on your system."

I think this is kind of nice because it flows into the next sentence about providing more information about `/etc/letsencrypt/live.` An alternative more like what you have here would be:

"...on your system. You must use the ``certonly`` command to get your certificate because as explained above, Certbot is unable to install your certificate from inside Docker."